### PR TITLE
Removed 2 newline characters and added style-errors.txt in .gitignore

### PR DESCRIPTION
**Pull request by Bryce-3D**
Removed 2 newline characters and added style-errors.txt in .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,5 @@ src/history/vacancy_history/data/cleaned
 src/history/vacancy_history/data/raw
 src/web/static/pages
 
-
-
 .DS_store
+style-errors.txt
```
Comment (lunaflight) [inline .gitignore:17]: You shouldn't include this in `.gitignore`. There is nothing in the source code that generates this text file, and leaves me confused. If this is a temporary text file which was used when you were piping the output of `ruff check` into, then do not include it when you use `git add` to stage it.

**Review by lunaflight — CHANGES_REQUESTED** [review]
Do read my comments on `style-errors.txt`.

File: .gitignore (hunk 1 above)
Comment (Bryce-3D) [reply .gitignore:17]: Oke noted. For reference I run the checkstyle by doing
`python -m ruff check > style-errors.txt`

**Comment by lunaflight** [comment]
Already covered by #9.

**Pull request closed without merging**